app/web/features/communities/events/EventCard.tsx

```diff
@@ -78,9 +78,15 @@ export const EVENT_CARD_TEST_ID = "event-card";
 export interface EventCardProps {
   event: Event.AsObject;
   className?: string;
+  // Optional formatter to override attendees count text (used by dashboard to avoid loading COMMUNITIES)
+  attendeesCountFormatter?: (count: number) => string;
 }
 
-export default function EventCard({ event, className }: EventCardProps) {
+export default function EventCard({
+  event,
+  className,
+  attendeesCountFormatter,
+}: EventCardProps) {
```

Comment:
This translation was the only one from the `COMMUNITIES` namespace loading on the dashboard. Dashboard was giving a warning on build about having too many strings and too big a size, so I made this translation dynamic so we could move a copy to the dashboard translations and remove the COMMUNITIES namespace from the translation SSR preloading.